uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1017.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1017 {
+
+  /**
+   * Migrating Phone Types and Deleting unwanted Phone Types
+   */
+  public function upgrade_1017() {
+    $this->migratePhoneTypes();
+    $this->relabelPhoneType("phone", "Landline");
+    $this->reservePhoneTypes(["phone", "mobile"]);
+    $this->deletePhoneTypes(["fax", "pager", "voicemail"]);
```

Comment:
Please add a space before the return statement. Also these phone types all begin with a capital letter on my civi installation, not sure why you have them all in lower case here.